app/main.py

```diff
@@ -1,3 +1,9 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    occurrences_count = 0
+    for letter_phrase in phrase.lower():
+        if letter_phrase == letter.lower():
+            occurrences_count += 1
+    return occurrences_count
```

Comment:
Please, try to implement this function in one line using chaining and built-in string methods to avoid variable allocation and a loop